### PR TITLE
Add implementation variant using push-pop abstraction

### PR DESCRIPTION
**Pull request by RossTate**
Not suggesting this should actually be merged. I just thought it might be helpful to have this as a reference for how this proposal could be implemented using the existing push-pop abstraction with special casing for only a few instructions (for now, just `select`).

### CODE REVIEW

File: proposals/relaxed-dead-code-validation/Overview.md [hunk 1]
```diff
@@ -145,42 +149,36 @@ Note that no attempt is made to make this efficient or cleverly reduce line-coun
...
-        pop_opd(I32)
-        push_opd(I32)
+      pop_opd(I32)
+      pop_opd(I32)
+      push_opd(I32)
 
     case (drop)
-      if (reachable)
-        pop_opd()
+      pop_opd()
```
Comment (conrad-watt) [inline proposals/relaxed-dead-code-validation/Overview.md]: I believe this will spuriously error for the code fragment

```
unreachable
drop
```

**Review by conrad-watt — COMMENTED** [review]
See single minor comment on `drop`.

It might be useful to have this alternative version to point at. Maybe we could put this in its own `.md` file which is linked to from the overview.

My main reason for not stating things in this way is that I was hoping to indicate roughly how I expected things to look if an implementation used a static/template parameter.

File: proposals/relaxed-dead-code-validation/Overview.md (hunk 1 above)
Comment (RossTate) [reply proposals/relaxed-dead-code-validation/Overview.md]: Ah, nice catch!

**Comment by RossTate** [comment]
Yeah, I thought having it has an alternative presentation might be the end result, but diffs are easier to review and I wasn't sure how we'd want to display an alternative (e.g. additional section in the overview or additional file).

**Comment by conrad-watt** [comment]
My knee-jerk - make it a separate file in the same directory as the overview and link it next to the overview text "... alternatively the reachability check could be pushed inside the pop/push functions in many cases."

**Comment by RossTate** [comment]
Force-pushed the same code (with the fix for `drop`) but in a separate file now.

**Comment by conrad-watt** [comment]
LGTM

**Pull request merged**